### PR TITLE
Small improvements to loading the configuration file and the location…

### PR DESCRIPTION
**Pull request by constantine-nikolaou**
… of the generated file. Also added basic instructions to run the script after cloning the repo. Bassem will updating README file with mode details.

### CODE REVIEW

File: slackcollector/collector.py [hunk 1]
```diff
@@ -148,8 +149,12 @@ def print_out(self, message, type='INFO'):
     # Create a new Collector instance
     # and pass the configuration as a param
     collector_inst = Collector()
+
+    dir = os.path.dirname(__file__)
```
Comment (Link-) [inline slackcollector/collector.py]: I have a problem with those, I need them be inside the Collector class and inside the `load_config()` method. For unit testing purposes, for exception handling purposes and separation of concerns.

File: samples/config.example.yml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Collector Configuration
 storage:
-    data_dir: /tmp/data
+    data_dir: output
```
Comment (Link-) [inline samples/config.example.yml]: Maybe we change this `<output_directory>` ?

File: README.md [hunk 3]
```diff
@@ -1,3 +1,43 @@
...
+`brew install python3`
+
+#### Step 2
+
+Install needed packages
+
+`pip3 install nose`
+
+`pip3 install tox`
+
```
Comment (Link-) [inline README.md:22]: I'm testing the setup on a fresh vm, those might not be needed if the user doesn't want to run the unit tests.

File: slackcollector/collector.py (hunk 1 above)
Comment (constantine-nikolaou) [inline slackcollector/collector.py]: Sure thing.

File: samples/config.example.yml (hunk 2 above)
Comment (constantine-nikolaou) [inline samples/config.example.yml]: You mean leave it for the user to define the directory? By all means

**Pull request closed without merging**